lib/ContactInfo.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+export default React.createClass({
+  getDefaultProps() {
+    return {
+      cont: {
```

Comment:
Can't say enough how awesome it is to see you use demo data like this. Theming it makes it become an entirely new experience! Always good that you kept it super topical as well!